src/crawler_utils.js

```diff
@@ -210,10 +176,13 @@ exports.handleDetail = async (page, request, extendOutputFunction, subtitlesSett
  * @param {number} maxRequested
  * @param {boolean} isSearchResultPage
  * @param {object} input
+ * @param {string} requestUrl
  */
 
-const getBasicInformation = async (page, maxRequested, isSearchResultPage, input) => {
-    const { youtubeVideosSection, youtubeVideosRenderer, url, videoTitle, channelNameText, subscriberCount, canonicalUrl } = CONSTS.SELECTORS.SEARCH;
+const getBasicInformation = async (page, maxRequested, isSearchResultPage, input, requestUrl) => {
```

Comment:
Five parameters are quite a lot, and the order can be easily mixed up. I would either wrap the parameters in some context object so the order does not matter or refactor it into multiple more straightforward functions.